### PR TITLE
Make error message

### PR DESCRIPTION
**Pull request by letrungkien0210**
Hi @jakubknejzlik ,

Please review.

Thanks.

### CODE REVIEW

File: src/login.js [hunk 1]
```diff
@@ -29,8 +29,10 @@ class Login extends Component {
     this.handleSubmit = this.handleSubmit.bind(this);
     
     this.state = {
-      username: '',
-      password: ''
+      username: 'test@novacloud.cz',
+      password: 'test',
```
Comment (jakubknejzlik) [inline src/login.js]: Please delete those credentials

File: src/login.js [hunk 2]
```diff
@@ -46,27 +48,34 @@ class Login extends Component {
...
+    };
+    
     rp(options)
-      .then(token => {
-        localStorage.set("token", token);
+      .then(response => {
+        const token = response.access_token;
+        const token_type = response.token_type;
+        this.changeStatusGetTokenError(false);
+        localStorage.setItem("token", `${token_type} ${token}`);
```
Comment (jakubknejzlik) [inline src/login.js]: Please split this to:
```
const refresh_token = response.refresh_token
localStorage.setItem("access_token", token);
localStorage.setItem("refresh_token", refresh_token);
localStorage.setItem("token_type", token_type);
```

File: src/login.js [hunk 3]
```diff
@@ -173,10 +206,11 @@ Login.defaultProps = {
...
   authProviders: ["facebook", "google", "gitlab"],
   authUrl: "",
-  tokenUrl: "",
+  tokenUrl: "https://novacloud.eu.auth0.com/oauth/token",
   redirectUrl: "",
-  client_id: "",
-  scope: ""
+  client_id: "mhjoWWbu6ByAfXKt73jMqNMjoFpA8vyQ",
+  scope: "email",
+  audience: "https://novacloud.eu.auth0.com/api/v2/",
```
Comment (jakubknejzlik) [inline src/login.js]: Please remove these credentials (tokenUrl, client_id, audience)

**Review by jakubknejzlik — CHANGES_REQUESTED** [review]

File: src/login.js (hunk 2 above)
Comment (letrungkien0210) [reply src/login.js]: response.refresh_token
The response don't have the property.

**Comment by letrungkien0210** [comment]
hi @jakubknejzlik ,

I fixed after you had commented.
Please review.

Thanks.

File: src/login.js [hunk 4]
```diff
@@ -46,27 +48,35 @@ class Login extends Component {
...
+    
     rp(options)
-      .then(token => {
-        localStorage.set("token", token);
+      .then(response => {
+        const access_token = response.access_token;
+        const token_type = response.token_type;
+        this.changeStatusGetTokenError(false);
+        localStorage.setItem("access_token", access_token);
+        localStorage.setItem("token_type", token_type);
```
Comment (jakubknejzlik) [inline src/login.js:72]: Let's store the refresh_token too

**Review by jakubknejzlik — CHANGES_REQUESTED** [review]

File: src/login.js (hunk 4 above)
Comment (jakubknejzlik) [reply src/login.js:72]: I see your comment...nevermind :)

**Review by jakubknejzlik — APPROVED** [review]

**Pull request merged**